cli.js

```diff
@@ -23,6 +24,17 @@ if (!cli.input[0]) {
 	process.exit(1);
 }
 
+var listForks = null;
+if (cli.flags.forks) {
+	switch (cli.flags.forks) {
+		case 'y': case 'yes': case 'true': case 't': listForks = true; break;
+		case 'n': case 'no': case 'false': case 'f': listForks = false; break;
```

Comment:
Why didn't you go with [boolean options][1]? You wouldn't have to parse all these cases manually, and you would also get `--no-forks` for free.

[1]: https://github.com/substack/minimist/blob/4cf45a26b9af5f4ddab63107f4007485e576cfd3/test%2Fbool.js